rskj-core/src/main/java/co/rsk/mine/minGasPrice/EthCallXRProvider.java

```diff
@@ -0,0 +1,56 @@
+package co.rsk.mine.minGasPrice;
+
+import co.rsk.config.StableMinGasPriceSourceConfig;
+import org.ethereum.core.Transaction;
+import org.ethereum.core.TransactionBuilder;
+
+import java.util.List;
+
+import static co.rsk.mine.minGasPrice.ExchangeRateProviderFactory.XRSourceType.ETH_CALL;
+
+public class EthCallXRProvider extends ExchangeRateProvider {
+    private final String address;
+    private final String method;
+    private final List<String> params;
```

Comment:
@Vovchyk  I know that the specs defines address and data props to the eth_call, but I don't know if we should expect the miners to encode the data themselves, if we can just do it here.